tests/models/test_bedrock.py

```diff
@@ -581,3 +581,37 @@ async def test_bedrock_multiple_documents_in_history(
     assert result.output == snapshot(
         'Based on the documents you\'ve shared, both Document 1.pdf and Document 2.pdf contain the text "Dummy PDF file". These appear to be placeholder or sample PDF documents rather than files with substantial content.'
     )
+
+
+async def test_bedrock_group_consecutive_tool_return_parts(bedrock_provider: BedrockProvider):
+    """
+    Test that consecutive ToolReturnPart objects are grouped into a single user message for Bedrock.
+    """
+    import datetime
+
+    from pydantic_ai.messages import ModelRequest, ToolReturnPart
+    from pydantic_ai.models.bedrock import BedrockConverseModel
+
+    model = BedrockConverseModel('us.amazon.nova-micro-v1:0', provider=bedrock_provider)
+    now = datetime.datetime.now()
+    # Create a ModelRequest with 3 consecutive ToolReturnParts
+    req = ModelRequest(
+        parts=[
+            ToolReturnPart(tool_name='tool1', content='result1', tool_call_id='id1', timestamp=now),
+            ToolReturnPart(tool_name='tool2', content='result2', tool_call_id='id2', timestamp=now),
+            ToolReturnPart(tool_name='tool3', content='result3', tool_call_id='id3', timestamp=now),
+        ]
+    )
+    # Call the mapping function directly
+    _, bedrock_messages = await model._map_messages([req])  # type: ignore[reportPrivateUsage]
+    # There should be one user message with 3 tool results
+    assert len(bedrock_messages) == 1
```

Comment:
Can you please use the [inline-snapshot](https://15r10nk.github.io/inline-snapshot/latest/) feature here? That'd look something like:

```py
assert bedrock_messages == snapshot()
```

Then when you run with `pytest tests/models/test_bedrock.py::test_bedrock_group_consecutive_tool_return_parts --inline-snapshots=create`, it'll fill that in with the messages returned, so we can quickly see the entire intended result and get a clear pytest error if any of it changes in a later update.